Documentation/BasicOperators.md

```diff
@@ -46,41 +46,394 @@ code-style will be used.
+
+```Swift
+let (numbersSignal, numbersSink) = SignalProducer<AnyObject, NoError>.buffer(5)
+let (lettersSignal, lettersSink) = SignalProducer<AnyObject, NoError>.buffer(5)
+let (signal, sink) = SignalProducer<SignalProducer<AnyObject, NoError>, NoError>.buffer(5)
+
+signal
+    |> flatten(FlattenStrategy.Merge)
+    |> start(next: { println($0) })
+
+sendNext(sink, numbersSignal)
+sendNext(sink, lettersSignal)
+sendCompleted(sink)
+
+sendNext(numbersSink, 1)    // prints 1
+sendNext(lettersSink, "A")  // prints A
+sendNext(numbersSink, 2)    // prints 2
+sendNext(lettersSink, "B")  // prints B
+sendNext(numbersSink, 3)    // prints 3
+sendNext(lettersSink, "C")  // prints C
+```
+
+### Concatenating
+
+The `.Concat` strategy works by concatenating the `SignalProducer`s such that their values are sent in the order of the `SignalProducer`s themselves. 
+This implies, that values of a `SignalProducer` are only sent, when the preceding `SignalProducer` has completed
+
+```Swift
+let (numbersSignal, numbersSink) = SignalProducer<AnyObject, NoError>.buffer(5)
+let (lettersSignal, lettersSink) = SignalProducer<AnyObject, NoError>.buffer(5)
+let (signal, sink) = SignalProducer<SignalProducer<AnyObject, NoError>, NoError>.buffer(5)
+
+signal
+    |> flatten(FlattenStrategy.Concat)
+    |> start(next: { println($0) })
+
+sendNext(sink, numbersSignal)
+sendNext(sink, lettersSignal)
+sendCompleted(sink)
+
+sendNext(numbersSink, 1)    // prints 1
+sendNext(lettersSink, "A")  // nothing printed
+sendNext(numbersSink, 2)    // prints 2
+sendNext(lettersSink, "B")  // nothing printed
+sendNext(numbersSink, 3)    // prints 3
+sendNext(lettersSink, "C")  // nothing printed
+sendCompleted(numbersSink)  // prints A, B, C
+sendCompleted(lettersSink)
+```
+
 ### Switching
 
+The `.Latest` strategy works by forwarding only events from the latest input `SignalProducer`.
+
+```Swift
+let (numbersSignal, numbersSink) = SignalProducer<AnyObject, NoError>.buffer(5)
+let (lettersSignal, lettersSink) = SignalProducer<AnyObject, NoError>.buffer(5)
+let (signal, sink) = SignalProducer<SignalProducer<AnyObject, NoError>, NoError>.buffer(5)
+
+signal
+    |> flatten(FlattenStrategy.Latest)
+    |> start(next: { println($0) })
+
+sendNext(sink, numbersSignal)   // nothing printed
+sendNext(numbersSink, 1)        // prints 1
+sendNext(lettersSink, "A")      // nothing printed
+sendNext(sink, lettersSignal)   // prints A
+sendNext(numbersSink, 2)        // nothing printed
+sendNext(lettersSink, "B")      // prints B
+sendNext(numbersSink, 3)        // nothing printed
+sendNext(lettersSink, "C")      // prints C
+```
 
 ## Handling errors
 
+These operators are used to handle errors that might occur on a signal.
+
 ### Catching errors
-### Mapping errors
+
+The `catch` operator catches any error that may occur on the input `SignalProducer`, then starts a new `SignalProducer` in its place.
+
+```Swift
```

Comment:
I'd structure this example a bit differently to better showcase that catch is good at creating sensible defaults in the face of errors.